src/routers/task.js

```diff
@@ -18,27 +21,28 @@ router.post("/tasks", async (req, res) => {
 });
 
 router.get("/tasks", async (req, res) => {
-  try {
-    const tasks = await Task.find({});
-    res.send(tasks);
-  } catch (e) {
-    res.status(500).send();
+  //const { description, completed } = req.body;
```

Comment:
Don't get in the habit of leaving useless code commented out in your commits. If it isn't needed, just remove it.